src/main/resources/assets/paths/nih.yaml

```diff
@@ -1,14 +1,14 @@
 delete:
   summary: Delete eRA Commons Account
-  description: Removes only eRA Commons Account related information from the authenticated user requesting this action
+  description: Removes Linked Account information for the authenticated user
   tags:
     - User
-    - eRA Commons
     - Nih
   responses:
     200:
       description: Returns an empty response if the operation was successful
 post:
+  deprecated: true
```

Comment:
Doesn't this endpoint need to go away entirely?

Reply:
When we completely move to RAS, yes. If we remove it now, the prod eRA Commons auth path will be broken until the UI part of this effort is complete. See https://broadworkbench.atlassian.net/browse/DT-2116 for that work.